src/test/spec/retryable_writes/mod.rs

```diff
@@ -319,21 +320,41 @@ async fn mmapv1_error_raised() {
 #[cfg_attr(feature = "async-std-runtime", async_std::test)]
 async fn label_not_added_first_read_error() {
     let _guard = LOCK.run_exclusively().await;
-    assert!(label_not_added(false, 1).await);
+
+    let options = ClientOptions::builder().retry_reads(false).build();
+    let client = TestClient::with_additional_options(Some(options), false).await;
+
+    let req = VersionReq::parse(">=4.0").unwrap();
```

Comment:
Could you add a comment here explaining why this check is needed? I'm guessing it's something to do with when the server started adding the label to errors, but I'm not sure.

Also, would it be cleaner to just put the skip check in the helper function rather than duplicating it in each of these tests?